Libraries/WebSocket/RCTWebSocketExecutor.m

```diff
@@ -49,10 +49,21 @@ - (instancetype)initWithURL:(NSURL *)URL
 - (void)setUp
 {
   if (!_url) {
-    NSUserDefaults *standardDefaults = [NSUserDefaults standardUserDefaults];
-    NSInteger port = [standardDefaults integerForKey:@"websocket-executor-port"] ?: 8081;
-    NSString *URLString = [NSString stringWithFormat:@"http://localhost:%zd/debugger-proxy", port];
-    _url = [RCTConvert NSURL:URLString];
+#define AUTO_IP
+// Uncomment the following line to disable the AUTO_IP configuration of the debugger
+// websocket.  See also the AUTO_IP logic in AppDelegate.m
```

Comment:
Nit - could we do a block comment here? Like `/** **/`? We should keep this AppDelegate file clean and easy to read.